oai_agents/gym_environments/base_overcooked_env.py

```diff
@@ -83,6 +86,9 @@ def __init__(self, grid_shape=None, ret_completed_subtasks=False, stack_frames=F
         self.step_count = 0
         self.reset_p_idx = None
 
+        # set up the learner. Different typese of learner would receive different rewards.
+        self.learner = self._get_learner_instance(args.learner_type)
```

Comment:
I would get rid of the `_get_learner_instance` function and instead would just do:
```
self.learner = Learner(learner_typer = args.learner_type)
```